features/autoconsent.json

```
@@ -450,6 +450,10 @@
         {
             "domain": "katieloxton.com",
             "reason": "https://github.com/duckduckgo/privacy-configuration/pull/2765"
+        },
+        {
+            "domain": "servustv.com",
+            "reason": "FIXME"
```

```suggestion
            "reason": "https://github.com/duckduckgo/privacy-configuration/pull/2772"
```

Maybe this is a better reason?

---

Whoops, well spotted. Done.